core/src/test/java/io/questdb/cairo/CreateTableTest.java

```diff
@@ -161,6 +161,69 @@ public void testCreateTableAsSelectInheritsColumnIndex() throws Exception {
         assertColumnsIndexed("new", "s");
     }
 
+    @Test
+    public void testCreateTableFromLikeTableWithNoIndex() throws Exception {
+        assertCompile("create table y (s1 symbol)");
+        assertQuery("s1\n","select * from tab", "create table tab (like y)", null);
+    }
+
+    @Test
+    public void testCreateTableFromLikeTableWithIndex() throws Exception {
+        assertCompile("create table tab (s symbol), index(s)");
+        assertQuery("s\n", "select * from x", "create table x (like tab)", null);
+        assertColumnsIndexed("x", "s");
+    }
+
+    @Test
+    public void testCreateTableFromLikeTableWithMultipleIndexes() throws Exception {
+        assertCompile("create table tab (s1 symbol, s2 symbol, s3 symbol), index(s1), index(s2), index(s3)");
+        assertQuery("s1\ts2\ts3\n", "select * from x", "create table x(like tab)",null);
+        assertColumnsIndexed("x", "s1", "s2", "s3");
```

Comment:
We need to verify that index block capacity is also same as in the source table.

Reply:
added test case